rules/aip0131/method_rules_test.go

```diff
@@ -92,3 +96,158 @@ func TestResponseMessageName(t *testing.T) {
 		})
 	}
 }
+
+func TestHttpVerb(t *testing.T) {
+	// Set up testing permutations.
+	tests := []struct {
+		testName   string
+		verb       string
+		methodName string
+		msg        string
```

Comment:
Use `wantMsg` to be more informative.

Reply:
Leaving this as-is. I would rather do a more thorough pass at comparing problems.